ci-operator/step-registry/upi/conf/vsphere/platform-external/upi-conf-vsphere-platform-external-commands.sh

```diff
@@ -35,6 +35,8 @@ if [[ ${vsphere_portgroup} == *"segment"* ]]; then
   third_octet=$(grep -oP '[ci|qe\-discon]-segment-\K[[:digit:]]+' <(echo "${vsphere_portgroup}"))
 
   machine_cidr="192.168.${third_octet}.0/25"
+  netmask="255.255.255.128"
+  gateway="192.168.${third_octet}.1"
```

Comment:
This could be cleaned up since all jobs have been migrated to using the `subnets.json`. Up to you